lisa/tools/modinfo.py

```diff
@@ -59,7 +59,11 @@ def get_version(
             no_info_log=True,
             no_error_log=True,
         )
-        return get_matched_str(output, self._version_pattern)
+        for line in output.splitlines():
```

Comment:
Can it be fixed by add "re.M" on the pattern?

Reply:
yes, fixed by adding re.M, updated and tested